include/deal.II/grid/grid_tools.h

```diff
@@ -323,12 +338,21 @@ namespace GridTools
    * Should this function be provided, sensible results can only be expected
    * if all coefficients are positive.
    *
+   * @param[in] solve_for_position_field If set to <code>true</code>, the
+   * minimization problem is formulated with respect to the final vertex positions
+   * as opposed to their displacement. The two formulations are equivalent for
+   * the homogeneous problem (default value of @p coefficient), but they
+   * result in very different mesh motion otherwise. Since in most cases one will
+   * be using heterogeneity coefficient in displacement formulation, the default
+   * value of this parameter is <code>false</code>.
+   *
    * @note This function is not currently implemented for the 1d case.
    */
   template <int dim>
   void laplace_transform (const std::map<unsigned int,Point<dim> > &new_points,
                           Triangulation<dim> &tria,
-                          const Function<dim,double> *coefficient = 0);
+                          const Function<dim,double> *coefficient = 0,
+                          const bool solve_for_position_field = false);
```

Comment:
Can we rename it to something like ``solve_for_absolute_positions`` or ``solve_for_disclacements``?